sklearn/src/cblas/cblas_sscal.c

```diff
@@ -0,0 +1,183 @@
+/* ---------------------------------------------------------------------
+ *
+ * -- Automatically Tuned Linear Algebra Software (ATLAS)
+ *    (C) Copyright 2000 All Rights Reserved
+ *
+ * -- ATLAS routine -- Version 3.2 -- December 25, 2000
+ *
+ * -- Suggestions,  comments,  bugs reports should be sent to the follo-
+ *    wing e-mail address: atlas@cs.utk.edu
+ *
+ * Author         : Antoine P. Petitet
+ * University of Tennessee - Innovative Computing Laboratory
+ * Knoxville TN, 37996-1301, USA.
+ *
+ * ---------------------------------------------------------------------
```

Comment:
I got it from the ATLAS distribution. They are all taken from "{ATLAS}/src/blas/reference/level*". In the level1 directory I see a ATL_srefscal.c that looks to me very similar to the one on this PR. Is this the one you say is "older"?